### PR TITLE
Add TypeScript definition

### PR DESCRIPTION
**Pull request by braaar**
Let's get some types up in here :)

This PR adds a simple `d.ts` file which declares the type of the postal code object to be `Record<string, string>`

I considered setting up a literal type for the entire postal code object, so that you have an object that is indexed by a set of literals, not just `Record<string, string>`, but I'm not sure how much value it really adds. 

Some devs may prefer not having to work with literals here, since in order to get a string out of the object you have to assert your string keys as index literals before indexing the postal code object:

```ts
    const postalCodes = (await import('norway-postal-codes')).default;

    let myPostalCode: string;

    const anyCity = postalCodes[myPostalCode]; // type is any

    if (myPostalCode in postalCodes) {
      const literalIndex = myPostalCode as keyof typeof postalCodes; // type is "1001" | "1003" | "1005" | etc.

      const stringCity = postalCodes[literalIndex]; // type is string
    }

```

On the other hand, such a type would certainly be preferable to `Record<string, string>` if your postal code is actually typed as a literal, not simply a string, for example if you have set up some kind of validation/sanitation pipeline. 

I think making such a type would work if we add this to `make.js`:
```js
	write(
		'index.d.ts',
		`declare module "norway-postal-codes" { export default ${JSON.stringify(
			ret2
		)}; }`
	);
```

What are your thoughts on typing this module?

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+declare module "norway-postal-codes" {
+  const postalCodes: Record<string, string>;
+
+  export default postalCodes;
+}
```
Comment (sindresorhus) [inline index.d.ts:5]: Please follow this style guide: https://github.com/sindresorhus/typescript-definition-style-guide/tree/57a925bb5b3eb46a4666eab31e1c883a26b7bfb5

File: package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
 	"license": "MIT",
 	"repository": "sindresorhus/norway-postal-codes",
 	"funding": "https://github.com/sponsors/sindresorhus",
+	"types": "index.d.ts",
```
Comment (sindresorhus) [inline package.json:8]: You need to add `index.d.ts` to the `files` field too.

**Comment by sindresorhus** [comment]
I think `Record<string, string>` is good enough.

File: index.d.ts (hunk 1 above)
Comment (braaar) [reply index.d.ts]: I think it should be compliant with the style guide now

File: package.json (hunk 2 above)
Comment (braaar) [reply package.json]: Done. Also moved the `types` entry to after dependencies as recommended in the style guide

File: index.d.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+declare const norwayPostalCodes: Record<string, string>;
```
Comment (sindresorhus) [inline index.d.ts:1]: Needs a doc comment

Comment (braaar) [reply index.d.ts:4]: Added a basic doc comment. Not sure if there is any particular way to make docstrings for objects.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**